src/test/java/org/opensearch/security/authtoken/jwt/JwtVendorTest.java

```diff
@@ -71,47 +168,524 @@ public void testCreateJwtWithRoles() throws Exception {
     }
 
     @Test(expected = Exception.class)
-    public void testCreateJwtWithBadExpiry() throws Exception {
```

Comment:
Seems like we are dropping coverage in some places? Did you mean to remove the bad expiry tests etc.? Is there somewhere else they have been moved to?